ci-operator/config/openshift/origin/openshift-origin-main.yaml

```diff
@@ -673,6 +673,18 @@ tests:
       - observers-resource-watch
     workflow: openshift-e2e-gcp
   timeout: 5h0m0s
+- always_run: false
+  as: e2e-gcp-ovn-usernamespace
```

Comment:
This can happen after this merges but you should likely go add detection for the Suite variant based on -usernamespace in the job name here: https://github.com/openshift/sippy/blob/main/pkg/variantregistry/ocp.go#L435

Note that this will keep your job off the component readiness board until it is opted in via a change to the default views. As such it will not be monitored unless someone on your team is explicitly monitoring it. If the pass rate is sufficient you should put up a PR to include it in the default view and we'll see if the org is willing to approve, at which point it's part of the blocking signal for a release. More details in that coming in this PR: https://github.com/openshift/ci-docs/pull/516/files

Reply:
Actually you might need to be careful with merge order across three prs. If you were to merge the job for a suite that doesn't exist, without the sippy PR, you could flip component readiness red eventually. I'd recommend you merge the origin PR, then this job, then the sippy PR.

Reply:
I think we will merge this one first, and if it shows in comp readiness we'll deal with that later.

However I noticed this is adding a presubmit on origin, I don't think that's what we actually want here, we're not worried about origin breaking your feature, we're worried about ocp breaking your feature itself. As such we should do a generic periodic run on a schedule. Presubmit also would not be able to feed signal to component readiness etc.

Add a regular periodic job instead and set the schedule appropriate for what you would like.

Reply:
Thank you!
I think if we put the periodic job, o/release is the best place, but there's no 4.20 jobs defined yet. They are going to be updated soon like https://github.com/openshift/release/pull/58747, right? Should I wait for the update?

Also I think we still want the presubmit job in o/origin to test the change. Do you think still we should remove the job in origin?

Reply:
I personally don't think we need to keep the origin presubmit job.